project_sprint_4

g-revert opened the pull request.

g-revert committed:
project_sprint_4

g-revert committed:
project_sprint_4

egorvas commented on .idea/.gitignore:
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
**Можно лучше:** Рекомендую добавлять папку .idea в .gitignore всего репозитория. По сути файлы в папке .idea это твои локальный файлы и не нужны для работоспособности проекта, но при этом могут приватные данные.

egorvas commented on src/test/java/TestCases.java:
```diff
@@ -0,0 +1,65 @@
+    }
+//Тестовый сценарий по выпадающему списку
+    @Test
+    public void test() {
```
**Нужно исправить:** это очень плохое название для теста. Название должно показывать, что именно проверяется в тесте.

egorvas commented on src/test/java/TestCases.java:
```diff
@@ -0,0 +1,65 @@
+
+        scooterOrderTest.waitForLoading();//ждем загрузки страницы
+        scooterOrderTest.clickOnCookies();//клик по кукам
+        scooterOrderTest.clickOnFirstOrderButton();//кликаем по кнопке Заказать
```
**Нужно исправить:** обрати внимание на пункт задания "Обрати внимание, что есть две точки входа в сценарий: кнопка «Заказать» вверху страницы и внизу. " Тебе нужно иметь два отдельных теста, с двумя отдельными кнопками заказа. Рекомендую использовать параметризацию, чтобы сократить кол-во кода.

egorvas commented on src/test/java/TestCases.java:
```diff
@@ -0,0 +1,65 @@
+        scooterPageTest.clickOnDropDownButton();//кликаем на выпадающий список
+        scooterPageTest.waitForLoading();//ждем появления списка
+        String actualText = scooterPageTest.getDropListText();//получаем текст в выпадающем списке
+        assertEquals("Текст не совпадает", EXPECTED_DROP_TEXT, actualText);//проверяем текст
```
**Нужно исправить:** тебе нужно проверить все кнопки элементов FAQ. Рекомендую каждую кнопку Faq вынести в отдельный тест и чтобы не копипастить много кода воспользоваться параметризацией.

g-revert committed:
project_sprint_4
Reworked the classes, added parameterized test for drop list texts, added the second flow of ordering the scooter in ScooterOrderTest.

g-revert committed:
project_sprint_4
Reworked the classes, added parameterized test for drop list texts, added the second flow of ordering the scooter in ScooterOrderTest.

g-revert committed:
project_sprint_4
Removed Assert method from MainPage class

g-revert committed:
project_sprint_4
Removed Assert method from MainPage class

g-revert committed:
project_sprint_4
Finalized

Aydinv13 committed:
Final
